### ISSUE TITLE
Place newly released Jakarta Starter prominently on main page

### ISSUE THREAD

**Issue by geziefer** [jakartaee/jakarta.ee#1675]
The Jakarta Starter (https://jakarta.ee/), which is available as a decent 2.0 relase, should be placed prominently on the main page, maybe next to the Download-Button.
That way, people can quickly start coding, the same as they have it on microprofile.io or quarkus.io for example.

**Comment by ivargrimstad** [comment]
The URL to the starter is https://start.jakarta.ee

**Comment by geziefer** [comment]
Thanks Ivar, of course, had the wrong url in clipboard.

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by oliviergoulet5** [comment]
Ideally we would add it in the header just like microprofile.io as it is quite nice. However, the header is already packed with links and wouldn't be able to fit another button as there is a width constraint of 1170px.

What we can do instead is add it to the jumbotron beneath the subtitle. I surrounded the region I'm speaking of in red:

![image](https://user-images.githubusercontent.com/17685861/236892594-764307a1-9f0b-47b1-92c5-28dd5720cc67.png)

Although this means it won't be available on every page but I presume that would be fine.

We can also put a download button in the same region. That way the Jakarta Starter won't appear to be the primary call to action. 

Any thoughts?

**Comment by TanjaObradovic** [comment]
Looks good to me! @shabnammayel please comment when you get a chance

**Comment by shabnammayel** [comment]
Is it possible to reduce the margin a bit to add a button in the navigation bar for the starter? We are working on developing more material for the starter and I think it deserves its own button on the navigation bar. When we revamped the website, we tried to stay away from adding buttons in the jumbotron 

<img width="1440" alt="Screen Shot 2023-05-12 at 7 37 20 AM" src="https://github.com/jakartaee/jakarta.ee/assets/51449073/0a58830f-b1c6-4c0d-8a99-40073eaa6400">

**Comment by chrisguindon** [comment]
There is not a lot of room here to add more buttons. I wouldn't play with the container padding since the header won't be aligned with the rest of the page after that. It will also impact how the page renders on mobile.

An idea would be to add a "Starterkit" button below the download button to avoid taking up more horizontal space:

![Screenshot 2023-05-16 103840](https://github.com/jakartaee/jakarta.ee/assets/893942/747d07c5-8d16-427c-8236-1afbbe75d5c1)

**Comment by oliviergoulet5** [comment]
We can also make the navbar take full width. This means it will no longer be centered along with the page content:

![wide](https://github.com/jakartaee/jakarta.ee/assets/17685861/75707de6-9a76-46b1-af21-6a1654ea3597)

Let me know which direction we want to take

**Comment by shabnammayel** [comment]
If we are able to make the navbar take full width, can we make the page content also take full width. it looks odd to make navbar full width and the content in the centre! the other option also doesn't look good to add another Botton below the download button! we want to create a operate button next to download button with a different color to stand out.

**Comment by chrisguindon** [comment]
> If we are able to make the navbar take full width, can we make the page content also take full width. 

We can make the content full width but not sure it's going to work with the look and feel of the website. 

@oliviergoulet5 Can you create a preview link where we switch from .container to .container-fluid so that we can see a live demo to help us decide if we should pursue this?

> the other option also doesn't look good to add another Botton below the download button! we want to create a operate button next to download button with a different color to stand out.

I understand my proposal is not perfect but our options are limited here... We don't have a lot of room in the header section for more links or buttons. We are trying to come up with creative solutions to make this work.


@shabnammayel  Can you reach out to Excentric and ask them if they can provide a solution to this problem If we can't agree to move forward with my proposal or @oliviergoulet5 proposal.?

**Comment by shabnammayel** [comment]
Thanks Chris! yes would be great to see a preview with full content and navbar view. @excentrickristy WDYT?

**Comment by oliviergoulet5** [comment]
@shabnammayel Here is a demo of container-fluid: https://deploy-preview-1680--jakartaee.netlify.app/

The demo's layout breaks at medium screen sizes. Let's just focus on the larger screen size for the demo.

**Comment by excentrickristy** [comment]
I think it looks good, but do you think it would be worth adding a max width on the wrapping container for larger screens @oliviergoulet5 ? Maybe just max-width : 1920px?

**Comment by oliviergoulet5** [comment]
@excentrickristy Yes adding a max-width of 1920px would be good.

**Comment by chrisguindon** [comment]
> Thanks Chris! yes would be great to see a preview with full content and navbar view. @excentrickristy WDYT?

@oliviergoulet5 I thought @shabnammayel asked to make the content fluid as-well? Did we change our mind on that?

**Comment by oliviergoulet5** [comment]
I missed that part, I'll create second preview with all content container-fluid. 

But @excentrickristy's recommendation with max-width is probably the ideal solution. We might just need to increase our container size after-all. I'll create a preview for that too.

**Comment by excentrickristy** [comment]
I would be careful about extending the content area beyond 1400px. We have found that's the max-width that keeps the content readable on the screen. Past that, if there is full width text it's hard to follow from line to line.

**Comment by chrisguindon** [comment]
> But @excentrickristy's recommendation with max-width is probably the ideal solution. We might just need to increase our container size after-all. I'll create a preview for that too.

I agree, I think the page will look better if it's not using 100% of the screen on a large desktop + for the reasons that @excentrickristy just shared in her last comment regarding readability.

**Comment by shabnammayel** [comment]
+1 to @excentrickristy to go with max width of 1920px. Do we have a new preview link @oliviergoulet5 or the one you shared above is the latest one? thanks

**Comment by shabnammayel** [comment]
My only thought is that it might be best to switch the placement of the download and starter button and switch the colours as well! make starter button orange and Download black and white. what do you think @ivargrimstad & @TanjaObradovic

**Comment by excentrickristy** [comment]
Mocked up a few options in the console for you, because I felt like the black is out of place/off-brand.

<img width="417" alt="image" src="https://github.com/jakartaee/jakarta.ee/assets/73264129/3ba0765b-e326-4e1d-9a8f-7a455feb273b">

<img width="370" alt="image" src="https://github.com/jakartaee/jakarta.ee/assets/73264129/b76b909e-ab2f-431f-afde-61c90aa296f0">

<img width="409" alt="image" src="https://github.com/jakartaee/jakarta.ee/assets/73264129/c488ef8d-f1d8-477f-9344-7f09f2f786bf">

**Comment by geziefer** [comment]
+1 for #3 with the orange Starter button.
Satisfies quite nicely my original requests.
Alex

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by shabnammayel** [comment]
+1 to @geziefer thanks @excentrickristy I like the starter in the orange button and white font the best! lets move forward with that

**Comment by OndroMih** [comment]
I like the #3 option the best too, very well done!

I just think that the word "Starter" might not be the best for newcomers or those that don't know that something like that exists and what it is. What about using "Get Started", "Start with Jakarta EE" or just "Start"? Or "Start Coding", which is used on the https://quarkus.io/ website for example...

**Comment by shabnammayel** [comment]
@oliviergoulet5 let's move forward with the option #3 then! For now we will use the word starter as we will need to add subpages for some of the starter articles that are now in works and also a guide for using the starter which we will probably have to call that one the Get Started!

**Comment by oliviergoulet5** [comment]
@shabnammayel Here is a preview of the navbar with the orange starter button: https://deploy-preview-1680--jakartaee.netlify.app/.

**Comment by excentrickristy** [comment]
Hi @oliviergoulet5, 
Can I suggest a change to the link padding for both the starter and the download button? padding: 8px 15px make the button look more.. buttony :)

**Comment by geziefer** [comment]
> @shabnammayel Here is a preview of the navbar with the orange starter button: https://deploy-preview-1680--jakartaee.netlify.app/.

Yes, looks nice, let's go for it!

**Comment by shabnammayel** [comment]
Thanks @oliviergoulet5 LGTM!
